src/test/resources/hudson/plugins/ec2/UnixData.yml

```diff
@@ -12,6 +12,11 @@ jenkins:
             type: "T2Micro"
             remoteFS: "/home/ec2-user"
             mode: "NORMAL"
+            spotConfig:
+                fallbackToOndemand: true
```

Comment:
Formatting looks off, the amiType block below is using 2 spaces for indent

Reply:
Oups, I compared the indentation with the "command prefix/suffixes" just below, I missed there was an extra key in between...